hw/dv/tools/dvsim/sim.mk

```
@@ -127,7 +127,7 @@ ifneq (${sw_images},)
 						if [[ $$artifact == *.bin && \
 							-f "$$(echo $${artifact} | cut -d. -f 1).elf" ]]; then \
 							cp -f "$$(echo $${artifact} | cut -d. -f 1).elf" \
-								$${run_dir}/$$(basename "$${artifact%.bin}.elf"); \
+								$${run_dir}/$$(basename -s .bin "$${artifact%.*.bin}").elf; \
```

if you want to use the `-s` option for the `basename` command, instead of the `%.bin` notation, then I think this be cleaner no?
```suggestion
								$${run_dir}/$$(basename -s .bin $${artifact}).elf; \
```

---

Sure, done.